.gitignore

```
@@ -66,3 +66,6 @@ logs/
 
 # Python cache #
 __pycache__
+
+# Node Modules #
+node_modules
```

Remember to end each file with a newline - this is for compatibility with unix features
```suggestion
node_modules

```